.pre-commit-config.yaml

```diff
@@ -56,19 +56,19 @@ repos:
   hooks:
   - id: flake8
     exclude: "^docs/"
-- repo: https://github.com/rhysd/actionlint
-  rev: v1.6.8
-  hooks:
-  - id: actionlint-docker
-    args:
-    - -ignore
-    - 'SC2155:'
-    - -ignore
-    - 'SC2086:'
-    - -ignore
-    - 'SC1004:'
-    - -ignore
-    - 'SC2193:'
+#- repo: https://github.com/rhysd/actionlint
```

Comment:
Mb we should enable this back?

Reply:
Sorry, my mistake: `actionlint` doesn't work on my M1 mac (no arm image available), so I disable it in order to run local linting of the code. I'll fix it now.